testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -1320,7 +1320,7 @@ void testCastStringToDateTime(CastType castType, SqlOperatorFixture f) {
     f.checkCastToString("DATE '1945-2-24'", null, "1945-02-24", castType);
 
     f.checkScalar("cast('1945-02-24' as DATE)", "1945-02-24", "DATE NOT NULL");
-    f.checkScalar("cast(' 1945-02-04 ' as DATE)", "1945-02-04", "DATE NOT NULL");
```

Comment:
Is it possible to add new unit tests instead of changing existing test cases?

Reply:
I am afraid we have to change the tests because they are wrong.